app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```

Comment:
- [ ] Nice work setting up devise, has a requirement kindly modify the views in devise for the registration and login  to match the wireframes and their styling.

### Your View
![wire-yours](https://user-images.githubusercontent.com/48371156/217953020-abf69e25-1df5-4c79-9887-9607e22a7c04.png)

### Wireframe
![image](https://user-images.githubusercontent.com/48371156/217953062-feb1015e-85fb-494a-b4f9-4402b42c04a8.png)

Reply:
Thank you so much for your feedback, will post my updates.